pdp/handlers.go

```diff
@@ -488,14 +488,22 @@ func (p *PDPService) handleGetProofSet(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Step 6: Prepare the response
+	// Step 7: Prepare the response
+	// Use 0 to indicate uninitialized proof set (no challenge epoch set yet)
+	// This maintains compatibility with SDK expectations
+	epochValue := int64(0)
+	if nextChallengeEpoch != nil {
+		epochValue = *nextChallengeEpoch
+	}
+
 	response := struct {
 		ID                 uint64      `json:"id"`
 		Roots              []RootEntry `json:"roots"`
 		NextChallengeEpoch int64       `json:"nextChallengeEpoch"`
 	}{
 		ID:                 proofSet.ID,
-		NextChallengeEpoch: nextChallengeEpoch,
+		NextChallengeEpoch: epochValue,
+		Roots:              []RootEntry{}, // Initialize as empty array, not nil
```

Comment:
@LexLuthr I don't think the response payload is changing here, right?
Note: it would be _really nice_ if these were typed structs instead of anonymous.